ContinuousML/NQ_DR_4_10_20_Ideal_27/2_Training_Workflow/dvc.yaml

```diff
@@ -12,4 +12,4 @@ stages:
     - ../../../src/Model_Training/Model_Training.py
     - Reduced_Features.csv
     outs:
-    - keras-model/
+    -   keras-model/
```

Comment:
This will be a machine learning model using ONNX library. We will probably version control the onnx folder that contains the model rather than just a single file.